docker/dev-environment-quick-start-guide.md

```diff
@@ -113,3 +118,16 @@ Also, you need to add a `cookie_domain` to the `%server_options` the same file,
 );
 ```
 Once you are done building your environment, go to http://localhost:8080/
+
+
+### 7. Optional - Import full mongodb dump
+The default docker environnement contains only ~120 products. If you need a full db with more than 1 millions products, you can import mongodb dump (1.7GB).
+```console
+$ wget https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.tar.gz
+$ docker cp openfoodfacts-mongodbdump.tar.gz docker_mongodb_1:/data/db
+$ docker exec -it docker_mongodb_1 bash
```

Comment:
I think it might be better to use `docker-composr` as in line 60. (Simple to use the same style of commands.)

Reply:
Can someone familiar with docker make the corresponding change so that we can merge? Sorry I still haven't tried it yet.